src/Features/CSharp/Portable/ExtractMethod/Extensions.cs

```diff
@@ -18,11 +18,12 @@ namespace Microsoft.CodeAnalysis.CSharp.ExtractMethod
 {
     internal static class Extensions
     {
-        public static ExpressionSyntax? GetUnparenthesizedExpression(this SyntaxNode? node)
+        [return: NotNullIfNotNull("node")]
```

Comment:
>return: [](start = 9, length = 7)

Are we or are we not using `return:` in the roslyn code base? I thought I read a comment from you a few prs ago that we weren't going to use it anymore?

Reply:
➡️ I already omitted `parameterName:` from this line.